Jenkinsfile

```diff
@@ -510,19 +513,22 @@ pipeline {
             }
         }
         stage("Smoke Fp32") {
+            options {
+                retry(2)
+            }
```

Comment:
I guess this is already in develop. Please update the branch.

Reply:
Actually it's an addition (retry on parallel stages instead of on individual ones).

Reply:
@junliume If "instead" then shall we remove retries in individual stages?